packages/react/src/stories/CardGroup/CardGroup.mdx

```diff
@@ -9,7 +9,7 @@ import { CardGroup } from "../../components";
 Card Group displays a group of Cards.
 
 <Canvas>
-  <Story of={CardGroupStories.GraphicCardGroup} />
+  <Story of={CardGroupStories.FeatureCardGroup} />
```

Comment:
Why change the default story?

Reply:
Changed it to be in the same order as twig, in the twig its defaulting to feature card.